src/widgets/helper/ChannelView.cpp

```diff
@@ -1966,8 +1966,7 @@ void ChannelView::disableScrolling()
 
 void ChannelView::scrollUpdateRequested()
 {
-    const qreal dpi =
-        QGuiApplication::screenAt(this->pos())->devicePixelRatio();
+    const qreal dpi = this->devicePixelRatio();
```

Comment:
`devicePixelRatio()` on a Widget returns an integer, not a qreal. Use `devicePixelRatioF()`

After that change has been done, the fun testing can begin :smile:

Reply:
done!